msal-dotnet-articles/acquiring-tokens/desktop-mobile/linux-broker-net.md

```diff
@@ -0,0 +1,149 @@
+---
+# Required metadata
+# For more information, see https://review.learn.microsoft.com/en-us/help/platform/learn-editor-add-metadata?branch=main
+# For valid values of ms.service, ms.prod, and ms.topic, see https://review.learn.microsoft.com/en-us/help/platform/metadata-taxonomies?branch=main
+
+title:       Use MSAL.NET with an authentication broker on Linux
+description: "Learn how to use MSAL.NET with a broker on the Linux platform."
+author:      xinyuxu1026 # GitHub alias
+ms.author:   xinyuxu # Microsoft alias
+ms.service:  msal
+# ms.prod:   # To use ms.prod, uncomment it and delete ms.service
+ms.topic:    how-to
+ms.date:     05/05/2025
+---
+
+# Use MSAL.NET with an authentication broker on Linux
+
+
+> [!NOTE]
+> Microsoft single sign-on (SSO) for Linux authentication broker support is introduced with `Microsoft.Identity.Client` version v4.69.1.
```

Comment:
I think this note should be removed. Instead, add the following .NET CLI commands to the Prereqs section below with minimum required versions:

dotnet add package Microsoft.Identity.Client
dotnet add package Microsoft.Identity.Client.Broker